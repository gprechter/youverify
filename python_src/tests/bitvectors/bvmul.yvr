a, b, c, d, e, f, g: BV[16]
call a = bvmul(BV{0, 16}, BV{245, 16})
call b = bvmul(BV{0, 16}, BV{0, 16})
call c = bvmul(BV{1, 16}, BV{345, 16})
call d = bvmul(BV{45, 16}, BV{345, 16})
e = BV{10, 16}
call f = bvmul(e, e)
call g = bvmul(e, c)