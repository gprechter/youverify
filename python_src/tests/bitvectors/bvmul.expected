{
    "a": 0,
    "b": 0,
    "c": 345,
    "d": 15525,
    "e": 10,
    "f": 100,
    "g": 3450
}